### DISCUSSION TITLE
Version 1.95.1 released

### DISCUSSION THREAD

**Discussion by cyring** [cyring/CoreFreq#406] [Show and tell]
@inglor Hello

Tag [`1.95.1`](https://github.com/cyring/CoreFreq/releases/tag/1.95.1) is available, including a short change-log.

**Comment by Atemu** [comment]
Could you make use of GitHub's release feature for announcing new releases?

**Reply by cyring** [reply]
OK, bookmarking [Managing releases in a repository](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository)

**Reply by cyring** [reply]
Now moving to #407

**Reply by Atemu** [reply]
Great, thanks!

**Reply by ich777** [reply]
@cyring can you maybe delete the Tag `show` over [here](https://github.com/cyring/CoreFreq/releases/tag/show), it was created by accident I think...?

**Reply by cyring** [reply]
> @cyring can you maybe delete the Tag `show` over [here](https://github.com/cyring/CoreFreq/releases/tag/show), it was created by accident I think...?

@ich777 Accident fixed. Thank you.
